alarm_backend/app.py

```diff
@@ -13,7 +16,32 @@
 
 
 @app.route("/alarms")
-def root():
+def get_all_alarms():
     """Get all alarms"""
+    all_alarm = []
     alarms = Alarm.query.all()
-    return alarms
\ No newline at end of file
+    for a in alarms:
+        all_alarm.append(serialize(a))
+    return jsonify(all_alarm)
+
+
+@app.route("/alarms/add", methods=['POST'])
```

Comment:
No need for the verb here
```suggestion
@app.route("/alarms", methods=['POST'])
```